app.js

```diff
@@ -56,6 +55,7 @@ app.use(basicAuth({
 
 app.use('/', index);
 app.use('/harvest', harvest);
+app.use(bodyParser.json());
```

Comment:
@jeffmcaffer to validate the payload in the webhook I needed the raw body so I put this back where it was before I moved it.

Reply:
makes sense